app/src/main/java/org/schabi/newpipe/util/PicassoHelper.java

```diff
@@ -133,28 +134,18 @@ public Bitmap transform(final Bitmap source) {
                                         .getDimension(R.dimen.player_notification_thumbnail_width),
                                 source.getWidth());
 
-                        final Bitmap result = Bitmap.createScaledBitmap(
+                        final Bitmap result = BitmapCompat.createScaledBitmap(
                                 source,
                                 (int) notificationThumbnailWidth,
                                 (int) (source.getHeight()
                                         / (source.getWidth() / notificationThumbnailWidth)),
+                                null,
                                 true);
 
-                        if (result == source) {
-                            // create a new mutable bitmap to prevent strange crashes on some
-                            // devices (see #4638)
```

Comment:
You should not ignore this comment. See #4638

Reply:
Oh, sorry, I didn't understand. I would rather not remove the workaround, since it caused many severe problems for users. But let's first ask the ones having problems to test themselves.